main/src/ca/uwaterloo/flix/language/fmt/FormatEff.scala

```diff
@@ -22,7 +22,7 @@ object FormatEff {
 
   def formatEff(eff: Type)(implicit flix: Flix): String = eff match {
     case Type.Cst(TypeConstructor.Pure, _) => "Pure"
-    case Type.Cst(TypeConstructor.EffUniv, _) => "IO"
+    case Type.Cst(TypeConstructor.EffUniv, _) => "Impure"
```

Comment:
I think we have to call top for "Univ"or something like that.

Reply:
Calling it impure, while perhaps correct, would be terribly confusing. Especially given early documentation. So I think a new word is better and I would hope that in practice we would never see "Univ" actually printed.